docs/components/custom_components.md

```diff
@@ -0,0 +1,115 @@
++          fields: ['title', 'text'],
++          find_answers: true,
++          max_answers_per_document: 1
++        },
++        _return: ['document_passages', 'extracted_metadata']
++      }}
++    >
+       <AppView />
+     </DiscoverySearch>
+   );
+```
+
+The overall details are explained in the article, but I did want to point out a couple things. First, `fields` should be set to reflect the data in your project/collection. This property tells Discovery in which document fields to look for answers. Second, we're setting `_return` to a set of top-level fields in the document results. Only the fields listed will be returned, cutting down on the response size (especially since we're not listing `text` and `html`, which can be quite large).
+
+(Note: Besides `overrideQueryParameters`, you can also update the query parameters by calling the `setSearchParameters` function on the `SearchApi` context.)
+
+Now let's build our custom component. The Discovery Components use [React Context](https://reactjs.org/docs/context.html) to pass data and give access to their API. The results of a search are provided in the `searchResponseStore` property of `SearchContext`. From within that, we'll find the most relevant `answers`, based on their `confidence` score.
+
+Copy the following code into `src/App.js`:
+
+```javascript
+// Show top 3 answers from within search results
+function Answers() {
+  // Use the context provided by <DiscoverySearch> to get search results
+  const {
+    searchResponseStore: { data: searchResponse }
+  } = useContext(SearchContext);
+
+  if (!searchResponse) {
+    return null;
+  }
+
+  const top3Answers = searchResponse.results
+    // extract document passages from documents in the results...
+    .reduce((acc, doc) => acc.concat(doc.document_passages), [])
+    // extract answers from the passages...
+    .reduce((acc, psg) => acc.concat(psg.answers), [])
+    // sort based on confidence score...
+    .sort((a, b) => b.confidence - a.confidence)
+    // and take the top 3
+    .slice(0, 3);
+
+  // render the answers as an ordered list
+  return (
+    <div>
+      <ol>
+        {top3Answers.map(answer => (
+          <li>{answer.answer_text}</li>
+        ))}
+      </ol>
+    </div>
+  );
+}
+```
+
+(Note: no styling provided for this component.)
+
+To make use of it, replace `<SearchResults>`:
+
+```diff
+             <div
+               className={`${settings.prefix}--col-md-6 ${settings.prefix}--search-app__facets-and-results__results`}
+             >
+-              {!isError ? <SearchResults /> : <p>An error occurred during search.</p>}
++              {!isError ? <Answers /> : <p>An error occurred during search.</p>}
+             </div>
+           </div>
+         </div>
+```
+
+## Write everything from scratch
+
+The example above uses a custom component alongside several existing Discovery Components. But you may wish to create everything from scratch. There's two ways to approach this.
```

Comment:
to avoid "above" and "alongside" which are sensory words (assume sighted users) i'd recommend: The previous example uses a custom component along with several ...
and slight reword suggestion for the next sentence:
But you might want to create everything from scratch. There are two ways to approach adding your own code.